Form/Type/AdminType.php

```diff
@@ -46,7 +46,16 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 $options['delete_options']['type_options']['translation_domain'] = $admin->getTranslationDomain();
             }
 
-            $builder->add('_delete', $options['delete_options']['type'], $options['delete_options']['type_options']);
+            $builder->add(
+                '_delete',
+                $options['delete_options']['type'],
+                array_merge(
+                    $options['delete_options']['type_options'],
+                    array('attr' => array(
+                        'class' => 'sonata-admin-type-delete-checkbox', ),
```

Comment:
I know right, styleCI forced this syntax(apparently it requires all arrays to have a comma after the last item)